Doras/src/app/carousel/carousel.component.html

```diff
@@ -0,0 +1,25 @@
+<div class='container-fluid' data-interval="false">
+    <div class='col-12'>
+        <ngb-carousel data-interval="false">
+            <ng-template ngbSlide>
```

Comment:
this should not be hardcoded, you should use the ngFor directive to loop over the list from your component!

Reply:
@lukemciKainos